presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/type/Timestamps.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.prestosql.type;
+package io.prestosql.plugin.base.type;
```

Comment:
We also have `Timestamps` in the SPI, so connectors now have two `Timestamps` classes available.
How do we decide which methods go into which class?

Reply:
Also, some methods from the class are specific to the engine (parsing literals),
and we should not move these.